desktop/src/main/java/bisq/desktop/main/portfolio/failedtrades/FailedTradesView.java

```diff
@@ -195,6 +205,38 @@ protected void activate() {
         sortedList.comparatorProperty().bind(tableView.comparatorProperty());
         tableView.setItems(sortedList);
 
+
```

Comment:
Remove extra newline.

Reply:
Current code in my workspace:

![Screenshot from 2023-02-28 16-29-16](https://user-images.githubusercontent.com/20237127/221900125-ecc569f8-829a-4002-9480-bc609df69f40.png)